resharper/resharper-unity/src/CSharp/Feature/Services/QuickFixes/CallGraph/ExpensiveCodeAnalysis/AddExpensiveComment/AddExpensiveCommentBulbAction.cs

```diff
@@ -0,0 +1,21 @@
+using JetBrains.Annotations;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.PerformanceCriticalCodeAnalysis.CallGraph;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.QuickFixes.CallGraph.ExpensiveCodeAnalysis.
+    AddExpensiveComment
+{
+    public class AddExpensiveCommentBulbAction : AddCommentActionBase
+    {
+        public AddExpensiveCommentBulbAction([NotNull] IMethodDeclaration methodDeclaration)
+            : base(methodDeclaration)
+        {
+        }
+
+        protected override string Comment =>
+            "//" + ReSharperControlConstruct.RestorePrefix + " " + ExpensiveCodeMarksProvider.MarkId;
```

Comment:
Add an extra space - I think this will generate `//ReSharper restore`. Let's make it `// ReSharper restore`